packages/bus/src/dispatchers/HttpCommandDispatcher.spec.ts

```diff
@@ -32,7 +38,15 @@ describe('HttpCommandDispatcher', () => {
       nock.activate();
     }
     spiedOptions = spy(options);
-    axiosDispatcher = new HttpCommandDispatcher(options);
+
+    when(mockedRetryStrategy.acquire(anyFunction())).thenCall(
+      (callback: (...args: unknown[]) => unknown) => callback()
+    );
```

Comment:
You have to add a few test cases to make sure that `acquire` will be called when it is expected